### PR TITLE
feat: german translations

### PR DESCRIPTION
**Pull request by scdanieli**
no-docs

### CODE REVIEW

File: erpnext/translations/de.csv [hunk 1]
```diff
@@ -9872,3 +9872,31 @@ Leave Type Allocation,Zuordnung Abwesenheitsarten,
...
+Accounts Receivable Ageing,Fälligkeit Forderungen,
+Accounts Payable Ageing,Fälligkeit Verbindlichkeiten,
+Budget Variance,Budgetabweichung,
+Based On Value,Basierend auf Wert,
+Restrict Items Based On,Artikel einschränken auf Basis von,
+Earnings & Deductions,Erträge & Abzüge,
+Is Process Loss,Ist Prozessverlust,
+Is Finished Item,Ist fertiger Artikel,
+Is Scrap Item,Ist Schrott,
+Issue a debit note with 0 qty against an existing Sales Invoice,Issue a debit note with 0 qty against an existing Sales Invoice,
```
Comment (barredterra) [inline erpnext/translations/de.csv]:
```suggestion
Issue a debit note with 0 qty against an existing Sales Invoice,Lastschrift mit Menge 0 gegen eine bestehende Ausgangsrechnung ausstellen,
```

File: erpnext/translations/de.csv [hunk 2]
```diff
@@ -8624,8 +8624,8 @@ Material Request Warehouse,Materialanforderungslager,
 Select warehouse for material requests,Wählen Sie Lager für Materialanfragen,
 Transfer Materials For Warehouse {0},Material für Lager übertragen {0},
 Production Plan Material Request Warehouse,Produktionsplan Materialanforderungslager,
-Sets 'Source Warehouse' in each row of the items table.,Legt &#39;Source Warehouse&#39; in jeder Zeile der Artikeltabelle fest.,
-Sets 'Target Warehouse' in each row of the items table.,"Füllt das Feld ""Ziel Lager"" in allen Positionen der folgenden Tabelle.",
+Sets 'Source Warehouse' in each row of the items table.,Legt in jeder Zeile der Artikeltabelle das &#39;Ausgangslager&#39; fest.,
+Sets 'Target Warehouse' in each row of the items table.,Legt in jeder Zeile der Artikeltabelle das &#39;Eingangslager&#39; fest.,
```
Comment (barredterra) [inline erpnext/translations/de.csv]:
```suggestion
Sets 'Source Warehouse' in each row of the items table.,Legt in jeder Zeile der Artikeltabelle das „Ausgangslager“ fest.,
Sets 'Target Warehouse' in each row of the items table.,Legt in jeder Zeile der Artikeltabelle das „Eingangslager“ fest.,
```

Comment (barredterra) [reply erpnext/translations/de.csv]: Not sure if this works, I hope `„“` are treated as normal characters 😅

Comment (scdanieli) [reply erpnext/translations/de.csv]: Yes, I've tested, seems fine.

**Review by barredterra — APPROVED** [review]

**Pull request merged**